app-foundation/README.md

```diff
@@ -0,0 +1,57 @@
+# terraform-example-foundation-app/app-foundation
+
+## Overview
+The `app-foundation` folder contains several distinct Terraform configurations that can supplement the CFT to deploy the necessary foundation for the Bank of Anthos example application. Each folder contains its own directory that must be applied separately, and in sequence, to correctly deploy the application. The necessary steps are outlined below for the relevant folders.
```

Comment:
In the overview we should link to example foundation and state that these configs are meant to fit into the framework defined by it.